testing/scripts/test_helpdesk_pw.py

```
@@ -2669,6 +2683,30 @@ def test_46_reporting_dashboard(page: Page):
+          page.locator('#swh-chart-status-empty').count() > 0,
+          "#swh-chart-status-empty not found")
+    check("reporting dashboard #254: trend empty-state element present in DOM",
+          page.locator('#swh-chart-trend-empty').count() > 0,
+          "#swh-chart-trend-empty not found")
+    try:
+        page.wait_for_function(
+            "() => { var el = document.getElementById('swh-chart-status-empty'); return el !== null && el.hidden === true; }",
+            timeout=5000,
+        )
+    except Exception:
+        pass  # element may not exist if Chart.js is slow; subsequent check records the result
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)